spring-cloud-aws-core/src/main/java/org/springframework/cloud/aws/core/io/s3/SimpleStorageResource.java

```diff
@@ -143,6 +145,10 @@ public URL getURL() throws IOException {
 				"/" + this.bucketName + "/" + this.objectName);
 	}
 
+	public URI getS3URI() throws URISyntaxException {
```

Comment:
Lets change method name to `getS3Uri` - when you look at other classes in Spring Framework whenever they have `URI` in name its spelled `Uri`.

`URISyntaxException` is rather impossible to happen right? We should rather catch it and throw `new RuntimeException` with explanation, similar to how it's done in `AmazonS3ClientFactory`.

Reply:
Ok. Fixed as requested.